src/router.ts

```diff
@@ -111,6 +143,23 @@ export class Router {
   */
   viewPortDefaults: any = {};
 
+  /**@internal */
+  catchAllHandler: (instruction: NavigationInstruction) => NavigationInstruction | Promise<NavigationInstruction>;
```

Comment:
not internal

Reply:
@davismj not public either ? I don't see it in the current `aurelia-router.d.ts`

Reply:
We were having problems generating the router.d.ts, which probably explains this.

Reply:
I'd vote for it to be like it is atm. We can always change it to public if necessary.